### DISCUSSION TITLE
Ignore mid-paragraph line breaks

### DISCUSSION THREAD

**Discussion by setrofim** [marp-team/marp#435] [Q&A] [answered]
Is there a way (a cmline flag, or a directive, etc) to get marp to behave similar to most other markdown-->html renderers with respect to handling line breaks? I.e. to ignore them inside paragraphs (unless inside a code block).

**Comment by yhatt** [comment, marked as answer]
In [Marp CLI](https://github.com/marp-team/marp-cli), you can set [markdown-it's constructor option `breaks`](https://markdown-it.github.io/markdown-it/#MarkdownIt.new:~:text=need%20HTML%20output.-,breaks,-%2D%20false.%20Set) to `false` by using Marp CLI configuration file.
https://github.com/marp-team/marp-cli#configuration-file

```javascript
// marp.config.js
module.exports = {
  // Set custom Marp Core constructor options
  // https://github.com/marp-team/marp-core#constructor-options
  options: {
    // markdown-it constructor options managed by Marp Core
    // https://github.com/markdown-it/markdown-it#init-with-presets-and-options
    markdown: {
      breaks: false
    }
  }
}
```

If you are using [Marp for VS Code](https://github.com/marp-team/marp-vscode), you can disable hardbreaks in the extension preference too. Set `markdown.marp.breaks` to `off`.

**Reply by setrofim** [reply]
Awesome! Exactly what I was looking for! Many thanks for the quick response.
